src/main/java/com/digirati/taxonomy/manager/lookup/persistence/model/SemanticRelationType.java

```diff
@@ -0,0 +1,29 @@
+package com.digirati.taxonomy.manager.lookup.persistence.model;
+
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.vocabulary.SKOS;
+
+public enum SemanticRelationType {
+    BROADER(SKOS.broader, SKOS.broaderTransitive),
+    NARROWER(SKOS.narrower, SKOS.narrowerTransitive),
+    RELATED(SKOS.related, SKOS.related),
+    IN_SCHEME(SKOS.inScheme, SKOS.inScheme),
+    HAS_TOP_CONCEPT(SKOS.hasTopConcept, SKOS.hasTopConcept),
```

Comment:
We need to store those relation types though if we want to be able to tell what concepts are part of a concept scheme. I'm happy to rename this enum to something that doesn't include the word "semantic", but I think splitting this out into multiple different enums, or storing the relationships in a different table is the wrong approach here

Reply:
Why do you think it's the wrong approach?

Reply:
I thought it was the wrong approach because the relationships are modelled in the exact same way - the only difference if we were to have two separate relation tables would be the type of the enum. However when I was tying it all together I had to work out whether to look in the concept or scheme tables for every entity involved in every relationship - which is expensive. I'll have to have a bit more of a think about it but I'm currently leaning towards thinking that splitting it into two would be better overall

Reply:
This is the only outstanding point left to action on this at the time of writing. I've had a bit more of a think about this and have realised that splitting it won't reduce the complexity of having to work out whether a relationship is between two concepts or between a scheme and a concept - it'll just shift move it out of the `SkosPersistenceService` and into the `SkosTranslator`. If anything I think it'll add to the complexity as anyone reading the code will have to now keep in mind an extra domain model and therefore an extra path through each bit of the `SkosPersistenceService`. So my suggestion here is to just drop the word "semantic" from this enum, as well as `ConceptSemanticRelationModel` and the corresponding table so that we're not overloading terms. What do you think?